src/fetcher.js

```diff
@@ -516,7 +516,8 @@ var Fetcher = function Fetcher (store, timeout, async) {
     } else {
       // See https://www.iana.org/assignments/link-relations/link-relations.xml
       // Alas not yet in RDF yet for each predicate
-      predicate = kb.sym(Uri.join(rel, 'http://www.iana.org/assignments/link-relations/'))
+      /// encode space in e.g. rel="shortcut icon"
+      predicate = kb.sym(Uri.join(encodeURIComponent(rel), 'http://www.iana.org/assignments/link-relations/'))
```

Comment:
Because any string which is not a URL is nighter here nor there, not a well-defined thing.   You can only compare URLs.  Things which random code in places converts into URLs, like fileames, people names, titles of documents, HRRP headers, link relations, whatever.... there is no standard for them, they onlly have well-defined properties in their own defined spaces like file system etc etc.   
When they all come together as URIs they are part of a single universal namespace.